src/b-st-ext.adoc

```diff
@@ -647,9 +647,9 @@ instructions that return the smaller/larger of two operands.
 
 ===== Sign- and zero-extension
 
-These instructions perform the sign-extension or zero-extension of the least significant 8 bits, 16 bits or 32 bits of the source register.
+These instructions perform the sign-extension or zero-extension of the least significant 8 bits or 16 bits of the source register.
```

Comment:
The zext.w is a 32-bit zero-extension; it is only a pseudo, but likely one of the most common usages of the add.wu instruction.

Reply:
Does that mean I should revert this change?

Reply:
I am pointing out that we have a 32bit zero-extension as one of our (pseudo-)instructions.

Strictly speaking, zext.w is not an instruction but a special-case of add.uw.
So strictly speaking, not listing 32 bit is correct — however, as a reader I would expect the 32 bit included (i.e., the change should be reverted).

It's a judgement call.

Reply:
It's a judgement call that a native speaker should make.

Reply:
Are you OK with accepting the approved change as is and allowing for any changes against the Chapter after it's merged into Unpriv?

Reply:
Let's let @aswaterman weigh in with a decision since he will have the best judgement for this specific issue.

Reply:
@aswaterman did approve this PR against the Bitmanip repo last night.  But I'm happy to see if Andrew has any additional comments.

Reply:
If I'm reading the document right, we're in a Zbb section of the document describing the sext/zext instructions in Zbb. zext.w is a pseudoinstruction from Zba so wouldn't belong in this section I think.

Reply:
Thanks @topperc , I am going to merge Bitmanip into Unpriv today, would you mind raising this as an issue against riscv-isa-manual after that happens?

Reply:
I'm not sure what issue you want me to raise. The zext.w pseudoinstruction is not listed in this section and shouldn't be listed in this section since its a Zbb section. Therefore only mentioning 8 and 16 bit here is correct given this structure.

Reply:
Oh my bad, I see your point now.

Reply:
I agree with @topperc that, taken in the context of this section within the Zbb spec, the change in the PR was correct.  We don't need to take any further action.